### PR TITLE
Add info on the state of development to the readme

### PR DESCRIPTION
**Pull request by eXpl0it3r**
Just adding a bit of info like we've done for the main repo and also adding @Marioalexsan as our new king 😉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,20 @@
 graphics, audio and network.
 It is originally written in C++, and this project is its official binding for .Net languages (C#, VB, ...).
 
+## State of Development
+
+Development is focused on the next major version in the `master` branch. No more features are planned for the 2.x release series.
+
+* The [`master`](https://github.com/SFML/SFML.Net/tree/master) branch contains work in progress for the next major version SFML.Net 3. As such it's considered unstable, but any testing and feedback is highly appreciated.
+* The [`2.6.0`](https://github.com/SFML/SFML.Net/tree/2.6.0) tag is the latest official SFML.Net release and will be the last minor release in the 2.x series.
+* The [`2.6.x`](https://github.com/SFML/SFML.Net/tree/2.6.x) branch will contain the latest bugfix work on SFML.Net 2.6.x, matching CSFMLs and SFMLs `2.6.x` branches.
```
Comment (ChrisThrasher) [inline README.md:13]:
```suggestion
* The [`2.6.x`](https://github.com/SFML/SFML.Net/tree/2.6.x) branch contains the latest bugfix work on SFML.Net 2.6.x, matching CSFMLs and SFMLs `2.6.x` branches.
```
This better matches the tense of previous bullet points.

File: README.md [hunk 2]
```diff
@@ -27,16 +37,16 @@ Of course, you can also find the SFML.Net API documentation in the SDK.
...
 
 For unsupported platforms or non-NuGet sources, you must have a copy of CSFML. CSFML can be compiled [from
 source](https://github.com/SFML/CSFML/) or downloaded from [the official release
 page](https://www.sfml-dev.org/download/csfml/). Also note that since CSFML depends on
 the main SFML project you also need all SFML runtime dependencies.
 
-Another dependency is the OpenTK library. This is required by the examples to run correctly.
-It is not required unless you plan on running the example programs that are included.
+Some of the example projects also use the OpenTK library to run correctly.
+But it is not required for SFML.Net itself.
```
Comment (ChrisThrasher) [inline README.md:49]:
```suggestion
Some of the example projects also require the OpenTK library to run correctly, but it is not required for SFML.Net itself.
```

File: README.md [hunk 3]
```diff
@@ -4,10 +4,20 @@
 graphics, audio and network.
```
Comment (Marioalexsan) [inline README.md:4]: Maybe this should be changed to mention that SFML.Net itself doesn't provide networking (i.e. we rely on .NET's own implementation)?

Comment (eXpl0it3r) [reply README.md:4]: > The network module is not provided in the SFML.Net binding as .NET provides superior networking support.

**Review by ChrisThrasher — APPROVED** [review]

**Review by Marioalexsan — APPROVED** [review]

**Pull request merged**